src/cascadia/Remoting/FindTargetWindowArgs.h

```diff
@@ -27,6 +27,7 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
     {
         GETSET_PROPERTY(winrt::Microsoft::Terminal::Remoting::CommandlineArgs, Args, nullptr);
         GETSET_PROPERTY(int, ResultTargetWindow, -1);
+        GETSET_PROPERTY(winrt::hstring, ResultTargetWindowName, L"");

```

Comment:
```suggestion
        GETSET_PROPERTY(winrt::hstring, ResultTargetWindowName);
```

This does the same thing without emitting an empty constant string. `GETSET_PROPERTY` uses `{}`, which for `winrt::hstring` is the empty hstring.